common/table.js

```diff
@@ -328,6 +348,7 @@
                     // open a new tab
                     var newRef = scope.vm.reference.table.reference.contextualize.entryCreate;
                     var appLink = newRef.appLink;
+                    // TODO: Necessary to check for existence of appLink?
```

Comment:
This is checking whether a `?` is in the string. If it is, use `&` instead (there's multiple params).